src/components/dialogs/RssDialog.vue

```diff
@@ -24,7 +24,7 @@
             @click="addRssItem"
             :title="$t('dialog.rss.add_feed')"
           >
-            <v-icon>mdi-link-plus</v-icon>
+            <v-icon>mdi-link-variant-plus</v-icon>
```

Comment:
reject

Reply:
I could potentially revert this. But since this icon is also used in the main add button, I thought a variant icon would look better.